src/osd/ReplicatedBackend.cc

```diff
@@ -789,7 +789,8 @@ void ReplicatedBackend::be_deep_scrub(
   ScrubMap::object &o,
   ThreadPool::TPHandle &handle)
 {
-  dout(10) << __func__ << " " << poid << " seed " << seed << dendl;
+  dout(10) << __func__ << " " << poid << " seed " 
```

Comment:
I'm going to leave it so it is there should anything be added to the end of that message.